buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -288,19 +307,18 @@
       "enabled": true,
       "continueOnError": true,
       "alwaysRun": true,
-      "displayName": "Copy Publish Artifact: BuildLogs",
+      "displayName": "Cleanup VSTS Agent",
       "timeoutInMinutes": 0,
       "task": {
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "CopyRoot": "$(PB_DockerCopyDest)/corefx",
-        "Contents": "**/*.log",
-        "ArtifactName": "BuildLogs",
-        "ArtifactType": "Container",
-        "TargetPath": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)"
+        "filename": "$(Build.SourcesDirectory)/corefx/Tools/msbuild.sh",
+        "arguments": "/p:AgentDirectory=$(Agent.HomeDirectory) /p:DoClean=$(PB_CleanAgent)",
+        "workingFolder": "$(Build.SourcesDirectory)/corefx/Tools/scripts/vstsagent/",
```

Comment:
I would slightly prefer having `cleanupagent.proj` in here somewhere, since the dir name isn't enough to tell what it's doing. (I don't have a preference either way on the full path being in the arguments (and removing the working dir) or just adding the filename.)